camus-kafka-coders/src/main/java/com/linkedin/camus/etl/kafka/coders/AvroMessageDecoder.java

```diff
@@ -0,0 +1,190 @@
+package com.linkedin.camus.etl.kafka.coders;
```

Comment:
Do we want to relocate this into an io.confluent package? The current package name is kind of misleading, and can potentially have a conflict if LI decides to release a generic Avro decoder with Camus at some point...